android/src/main/java/com/balthazargronon/RCTZeroconf/ZeroconfModule.java

```diff
@@ -118,8 +118,12 @@ protected void sendEvent(ReactContext reactContext,
     private class ZeroResolveListener implements NsdManager.ResolveListener {
         @Override
         public void onResolveFailed(NsdServiceInfo serviceInfo, int errorCode) {
-            String error = "Resolving service failed with code: " + errorCode;
-            sendEvent(getReactApplicationContext(), EVENT_ERROR, error);
+            if(errorCode == NsdManager.FAILURE_ALREADY_ACTIVE) {
```

Comment:
Missing space after the `if` statement.

Also, do we really want to resolve the service again if it has already been resolved? Shouldn't we just catch this case and do nothing?